legistar/events.py

```diff
@@ -125,9 +125,12 @@ def events(self, since_datetime):
             key = (api_event['EventBodyName'].strip(),
                    self.toTime(api_event['EventDate']).date(),
                    api_event['EventTime'])
-
-            web_event = web_results[key]
-            yield api_event, web_event
+            try:
+                web_event = web_results[key]
+                yield api_event, web_event
```

Comment:
I kinda like this idiom

```python
try:
    the thing you are worried about raising an exception
except Particular Exception:
    the thing do if you get the exception
else:
    the thing to do if you don't get the exception
```

it doesn't really matter for such a simple case as this, but sometimes you have complicated stuff to do if you *don't* throw the exception, and sometimes thos things can also throw exceptions. With this idiom it's clear what you are trying to keep an eye on.